security/security.c

```diff
@@ -86,6 +90,45 @@ static __initconst const char * const builtin_lsm_order = CONFIG_LSM;
 static __initdata struct lsm_info **ordered_lsms;
 static __initdata struct lsm_info *exclusive;
 
+/* static slots init */
```

Comment:
This comment doesn't tell the reader anything - I'd either expand it or delete it (I think in this case either is fine)